### PR TITLE
En 964 languages and labels

### PR DESCRIPTION
**Pull request by alessioscalici**

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

File: src/state/labels/actions.js [hunk 1]
```diff
@@ -64,3 +71,18 @@ export const createLabel = label => dispatch => (
...
+        response.json().then(() => {
+          dispatch(removeLabelSync(label));
+          resolve();
+        });
+      } else {
+        resolve();
+      }
+    });
+  })
+);
```
Comment (fedelocci) [inline src/state/labels/actions.js]: i this case if thare are error is not calling.
i should be like this?

export const removeLabel = label => dispatch => (
  new Promise((resolve) => {
    deleteLabel(label).then((response) => {
      
        response.json().then(() => {
if (response.ok) {
          dispatch(removeLabelSync(label));
          resolve();
} else {
dispatch(addErrors(json.errors.map(err => err.message)));
        resolve();
      }
        });
      
    });
  })
)

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Pull request merged**